lib/internal/modules/cjs/loader.js

```diff
@@ -1203,6 +1203,11 @@ function wrapSafe(filename, content, cjsModuleInstance) {
   } catch (err) {
     if (process.mainModule === cjsModuleInstance)
       enrichCJSError(err);
+    if (err.message.includes('await is only valid in async function')) {
```

Comment:
what happens if you plug in this code:

```js
function x() {
  await 1;
}
```

Reply:
```js
/Users/mylesborins/code/node/main/lol.js:2
  await 1;
  ^^^^^

SyntaxError: Top-Level await is only supported in ESM.
    at wrapSafe (internal/modules/cjs/loader.js:1005:16)
    at Module._compile (internal/modules/cjs/loader.js:1058:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1115:10)
    at Module.load (internal/modules/cjs/loader.js:954:32)
    at Function.Module._load (internal/modules/cjs/loader.js:795:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
```

Yeah, this is not good. Do you think there is a way to distinguish this without having to parse the entire file?